### ISSUE TITLE
A possible solution for the routing issue

### ISSUE THREAD

**Issue by wmertens** [AltSchool/ember-cli-react#43]
I'm just scoping the field, I'm somewhat of an Ember newbie, not using this project (yet), but I noticed that you say that `to-link` doesn't work, and it reminded me of this approach for generic links:

* put a click handler around the whole app
* on click, see if a link was clicked ([here's some example code](https://github.com/Yaska/redux-addressbar/blob/39314d0dc02c0807b9d7e60f131c289f5d49c9a7/src/index.js#L26-L64))
* if the link was one for this website, pass it to the routing mechanism

So in other words, the React component would just have to generate regular `<a href="..."/>` and it would be injected into Ember.
